streams-runtimes/streams-runtime-threaded/src/main/java/org/apache/streams/threaded/controller/ThreadingController.java

```diff
@@ -65,15 +66,37 @@
      * The threading controller
      */
     public static ThreadingController getInstanceLowPriority() {
-        return INSTANCE_LOW_PRIORITY;
+        synchronized (ThreadingController.class) {
```

Comment:
yeah, I understand.. I like to do it this way since it is performance neutral. so the caller can see what is going on better.